pkg/controller.v1/tensorflow/controller.go

```diff
@@ -118,7 +118,10 @@ func NewTFController(
 	tfJobInformerFactory tfjobinformers.SharedInformerFactory,
 	option options.ServerOption) *TFController {
 
-	tfjobscheme.AddToScheme(scheme.Scheme)
+	err := tfjobscheme.AddToScheme(scheme.Scheme)
```

Comment:
Yeah, I know that the code is updated by the code-generator. Then if we update it, we should add a err check to avoid undefined behaviour. Code here LGTM 

/lgtm